extract/raw_array_cohort_extract.py

```diff
@@ -99,18 +100,29 @@ def get_subselect(fq_array_table, samples, id):
       j = 1
       for samples in split_lists(partition_samples, 1000):
         id = f"{i}_{j}"
-        subs[id] = get_subselect(fq_vet_table, samples, id)
+        subs[id] = get_subselect(fq_vet_table, samples, id, extract_genotype_counts_only)
```

Comment:
should fq_vet_table be fq_array_table?

also, above, why are we splitting the samples by 1000?

Reply:
Yeah, I can change vet -> array

@kcibul Any thoughts on why this is splitting samples by 1000? Is there a limit somewhere?